### ISSUE TITLE
Is there any tricks in reproducing the result on AGENDA dataset?

### ISSUE THREAD

**Issue by KSXGroup** [UKPLab/plms-graph2text#1]
As you write in the paper, I concat the title, entities and relations as input of BART-base model and training it with the hyper-parameter you provided in agenda/finetune_graph2text.sh with fairseq. But I only get BLEU4=16.87. Did you add any additional information or use any trick to get the result on your paper ?

**Comment by KSXGroup** [comment]
Update, after some hyper-parameter tuning, I reached BLEU4=19.22 after 11 epoch with learning rate 5e-5.
However, there is still a gap between my result and your result in paper(BLEU4=22.01).

I find that you did not upload the data-preprocessing part and detailed result & output of AGENDA dataset. Can you upload it for reproduction ?

**Comment by linzhk** [comment]
> Update, after some hyper-parameter tuning, I reached BLEU4=19.22 after 11 epoch with learning rate 5e-5.
> However, there is still a gap between my result and your result in paper(BLEU4=22.01).
> 
> I find that you did not upload the data-preprocessing part and detailed result & output of AGENDA dataset. Can you upload it for reproduction ?

hello, I failed to start this experiment because of the lack of agenda/data/generate_input_agenda.py. Could you send the missing file to me if it is available? Thanks a lot if you do me favor

**Comment by KSXGroup** [comment]
> > Update, after some hyper-parameter tuning, I reached BLEU4=19.22 after 11 epoch with learning rate 5e-5.
> > However, there is still a gap between my result and your result in paper(BLEU4=22.01).
> > I find that you did not upload the data-preprocessing part and detailed result & output of AGENDA dataset. Can you upload it for reproduction ?
> 
> hello, I failed to start this experiment because of the lack of agenda/data/generate_input_agenda.py. Could you send the missing file to me if it is available? Thanks a lot if you do me favor

You can just follow his paper's instruction. Input the linear version of the graph after the concatenation of title and entities and it's quite easy to implement.  However, I implement integrate it this into fairseq framework for faster training, so it may not be  compatible with  your code...

**Comment by wutaiqiang** [comment]
@KSXGroup Could you please share the file after the preprocessing? I mean the preprocessed file could be used for finetuning directed.  Thank you~
